examples/asset/loading_screen.rs

```diff
@@ -52,15 +52,14 @@ fn asset_listening_system(
 fn setup_game(
     mut commands: Commands,
     asset_server: Res<AssetServer>,
-    mut loading_screen: Query<(&mut Style, &Children), With<plugin::LoadingScreen>>,
-    mut text_query: Query<&mut Text>,
+    loading_screen: Query<&Children, With<plugin::LoadingScreen>>,
+    mut text_query: Query<&mut Text, With<plugin::LoadingScreenContent>>,
 ) {
     // Change our default loading screen text to something more descriptive
-    for (mut _style, children) in loading_screen.iter_mut() {
-        for child in children.iter() {
-            if let Ok(mut loading_text) = text_query.get_mut(*child) {
-                loading_text.sections[0].value = "Loading \"FlightHelmet\"".to_string();
-            }
+    let children = loading_screen.single();
+    for child in children.first() {
```

Comment:
could you do something like
```rust
        if let Ok(mut loading_text) = text_query.get_mut(*cildren.first()) {
            loading_text.sections[0].value = "Loading \"FlightHelmet\"".to_string();
        }
```
If that doesn't work, at least use a let binding for the child. I feels really weird to loop on the result of `first()` which should always be a single element.

Reply:
Haha! I already did that locally, because clippy was not happy with me 😅